.travis.yml

```diff
@@ -2,6 +2,7 @@ language: java
 sudo: false
 jdk:
   - oraclejdk8
+  - oraclejdk9
```

Comment:
Maybe also add Java 10 since you added Java 10 specific changes in build.gradle?